doc/source/management/innodb_kill_idle_trx.rst

```diff
@@ -1,28 +1,52 @@
 .. _innodb_kill_idle_trx:
 
-========================
- Kill Idle Transactions
-========================
+======================
+Kill Idle Transactions
+======================
 
-This feature limits the age of idle |XtraDB| transactions. If a transaction is idle for more seconds than the threshold specified, it will be killed. This prevents users from blocking purge by mistake.
+This feature limits the age of idle |XtraDB| transactions. If a transaction is
+idle for more seconds than the threshold specified, it will be killed. This
+prevents users from blocking purge by mistake.
+
+In |Percona Server| :rn:`5.7.17-11` this feature has been re-implemented by
+setting a different socket read timeout value instead of periodically scanning
+the low |InnoDB| transaction list level.
 
 Version Specific Information
 ============================
 
   * :rn:`5.7.10-1`:
         Feature ported from |Percona Server| 5.6
+  * :rn:`5.7.17-11`:
+        Feature re-implemented
 
 System Variables
 ================
 
 .. variable:: innodb_kill_idle_transaction
-   
+
+   :version 5.7.17-11: Variable is now an alias of :variable:`kill_idle_transaction`
+   :scope: ``GLOBAL``
+   :config: ``YES``
+   :dyn: ``YES``
+   :vartype: ``INTEGER``
+   :default: 0 (disabled)
+   :unit: Seconds
+
+   To enable this feature, set this variable to the desired seconds wait until
+   the transaction is killed. **NOTE:** This variable has been deprecated and
+   it will be removed in future major release.
+
+.. variable:: kill_idle_transaction
+
+   :version 5.7.17-11: Variable implemented
    :scope: ``GLOBAL``
    :config: ``YES``
    :dyn: ``YES``
    :vartype: ``INTEGER``
    :default: 0 (disabled)
    :unit: Seconds
 
-   To enable this feature, set this variable to the desired seconds wait until the transaction is killed.   
+   To enable this feature, set this variable to the desired seconds wait until
```

Comment:
If non-zero, any idle transaction will be killed after being idle for this many seconds.